src/EnergyPlus/BranchInputManager.cc

```diff
@@ -607,65 +606,6 @@ namespace BranchInputManager {
 		return GetAirBranchIndex;
 	}
 
```

Comment:
The remaining changes will help to simplify code where air flow is dictated by the system and not the ductwork. Move forward with this.